src/app/core/config/config-fix.ts

```diff
@@ -644,9 +644,10 @@ export const defaultJsonConfig = {
           "components": [
             {
               "title": $localize`:Title inside a panel:School History`,
-              "component": "PreviousSchools",
+              "component": "ChildSchoolOverview",
               "config": {
                 "single": true,
+                "clickMode": "navigate",
```

Comment:
```suggestion
```
This is not working in the demo.

Reply:
:+1: thanks. forgot this here from testing, apparently